apps/workers/src/main.ts

```diff
@@ -154,11 +154,13 @@ syncInstitutionQueue.add(
  */
 sendEmailQueue.process('send-email', async (job) => await emailProcessor.send(job.data))
 
-sendEmailQueue.add(
-    'send-email',
-    { type: 'trial-reminders' },
-    { repeat: { cron: '0 */12 * * *' } } // Run every 12 hours
-)
+if (env.STRIPE_API_KEY) {
```

Comment:
lets also make sure there's not a default value in the workers .env file that would cause a false positive

Reply:
🤦  yeah you're right, I was thinking of the NX_STRIPE_SECRET_KEY which I'm not sure if even used anywhere 🤔

Reply:
Yeah, I saw that too... We're probably going to make some updates to the Zod schema once we figure out what each env variable is doing

Reply:
For sure, will leave that to another PR. LGTM!